components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -699,7 +699,7 @@ public static void addAutoscalingPolicy(AutoscalePolicyBean autoscalePolicyBean)
      * @throws RestAPIException
      */
     public static void addApplicationPolicy(ApplicationPolicyBean applicationPolicyBean) throws RestAPIException,
-            AutoscalerServiceInvalidApplicationPolicyExceptionException {
+            AutoscalerServiceInvalidApplicationPolicyExceptionException,AutoscalerServiceApplicationPolicyAlreadyExistsExceptionException {
```

Comment:
+1 for the suggestion, @pubudu538 would you mind sending another PR with proper code formatting?
Thanks